Gemfile.lock

```diff
@@ -99,7 +99,7 @@ GEM
     execjs (2.7.0)
     faker (2.17.0)
       i18n (>= 1.6, < 2)
-    faraday (1.7.1)
+    faraday (1.8.0)
```

Comment:
Are these additional changes to the Gemfile.lock necessary? Normally these are handled by dependabot PRs, so gem version updates (like this) are solely about versioning the components gem itself, and nothing else.

Reply:
Perhaps not but I'm finding that they are bumped each time I run `bundle install && yarn install` - but it does seem odd. I can't see any reason for the versions to be changed.

Reply:
The documentation does say to do that, but I think that's incorrect (I'll look at changing that). Meanwhile can I suggest that you revert these changes and only modify the `govuk_publishing_components` line in this file?

Reply:
No problem, just updated that now